pkg/imager/fyne.go

```diff
@@ -93,7 +93,13 @@ func (fg *FyneUI) listBaseImgs() ([]string, string) {
 	d.Hide()
 
 	if err != nil {
-		dialog.ShowError(err, fg.w)
+		var message string
+		if errors.Is(err, ErrNetworkConn) {
+			message = "Network connection error"
```

Comment:
We might also return something like: 

`Network connection error. Please ensure that you are connected to the internet correctly.`